impl/authentication/context.go

```diff
@@ -14,24 +14,25 @@
 
 // Package authentication implements authentication mechanisms.
 //
-// The Transparent Key Server is designed to be used by identity providers -
+// Key Transparency is designed to be used by identity providers -
 // IdP in OAuth parlance.  OAuth2 Access Tokens may be provided as
 // authentication information, which can be resolved to user information and
 // associated scopes on the backend.
 package authentication
 
-import (
-	"context"
-	"errors"
-)
+import "context"
 
-var (
-	// ErrMissingAuth occurs when authentication information is missing.
-	ErrMissingAuth = errors.New("auth: missing authentication header")
-)
+// ValidatedSecurity is the auth value stored in the Contexts.
+type ValidatedSecurity struct {
```

Comment:
Maybe this should be called SecurityContext, or the key renamed so they match.

Reply:
Renamed.

Reply:
Renamed